app/utils/EligibilityCookie.js

```diff
@@ -50,9 +50,9 @@ class EligibilityCookie {
         const cookieValue = JSON.stringify(json);
 
         if (req.protocol === 'https') {
-            res.cookie(eligibilityCookieName, cookieValue, {httpOnly: true, secure: true, expires: new Date(Date.now() + 172800)});
+            res.cookie(eligibilityCookieName, cookieValue, {httpOnly: true, expires: new Date(Date.now() + 172800), secure: true});
         } else {
-            res.cookie(eligibilityCookieName, cookieValue, {httpOnly: true});
+            res.cookie(eligibilityCookieName, cookieValue, {httpOnly: true, expires: new Date(Date.now() + 172800)});
```

Comment:
I think it would be better to define the expiry time (`172800`) in the config and use it from there in all the places.

Also we could reduce the code complexity of this conditional by creating the cookie params object before the conditional, setting `secure: true` only in the conditional and creating the cookie after the conditional

Reply:
Done